dashboard/app/controllers/home_controller.rb

```diff
@@ -45,13 +45,13 @@ def index
   end
 
   # Signed in: render home page
-  # Signed out: redirect to code.org
+  # Signed out: redirect to sign in
   def home
     if current_user
       init_homepage
       render 'home/index'
     else
-      redirect_to CDO.code_org_url
+      redirect_to '/users/sign_in'
```

Comment:
Also yes, Brendan, it's `new_user_session_path:
![image](https://user-images.githubusercontent.com/8405316/28985887-845bdc54-7919-11e7-8c7b-32e608e13715.png)

Neat trick: it takes a minute, but you can see all routes and helpers on localhost at http://localhost-studio.code.org:3000/rails/info/routes, or on the command line with `rails routes`.